cloudflare/custom_hostname.go

```diff
@@ -0,0 +1,172 @@
+package cloudflare
```

Comment:
This filename needs to follow the conventions of the other files. Check out the `cloudflare/` directory for guidance.